scripts/install

```diff
@@ -176,6 +163,14 @@ pushd "$final_path"
 	ynh_exec_warn_less ynh_exec_as $app bin/mattermost team add "$team_name" "$admin"
 popd
 
+#=================================================
+# SETUP LOGROTATE
+#=================================================
+ynh_script_progression --message="Configuring log rotation..." --weight=1
+
+# Use logrotate to manage application logfile(s)
+ynh_use_logrotate
```

Comment:
Will it also do the `chown $app -R $logs_path` part?

Reply:
Looks like not: https://github.com/YunoHost/yunohost/blob/b452838a1797641f82778e60f4e50e1722837eba/data/helpers.d/logrotate#L97